### PR TITLE
Possible fix by differentiating newer model floodlightmode APIs for coaxial API use on floodlights.

### PR DESCRIPTION
**Pull request by cchamilt**
It appears that some of the earlier models support Coaxial API, but not the floodlightmode.  Since the 26 earlier models appear to use LightingV2 OK, this explicitly associates the Coaxial API calls to the 4x models that have floodlightmode.

### CODE REVIEW

**Review by rroller — APPROVED** [review]

File: custom_components/dahua/__init__.py [hunk 1]
```diff
@@ -549,6 +552,10 @@ def is_flood_light(self) -> bool:
         m = self.model.upper()
         return m.startswith("ASH26") or "L26N" in m or "L46N" in m or m.startswith("V261LC") or m.startswith("W452ASD")
 
+    def supports_floodlightmode(self) -> bool:
+        """ Returns true if this camera supports floodlight mode """
+        return "W452ASD" in self.model.upper() or "L46N" in self.model.upper()
```
Comment (rroller) [inline custom_components/dahua/__init__.py:557]: I think one of my cams is a different model but supports the floodlight. Once I get home I'll try to check...

**Comment by gkwok1** [comment]
Any updates on this to fix floodlight issues with Amcrest ASH26 in v0.51?

File: custom_components/dahua/__init__.py (hunk 1 above)
Comment (gkwok1) [reply custom_components/dahua/__init__.py:557]: ASH26 flood light is still broken in 0.9.54. Anyway we can get this merged on the next release?

**Comment by skooj** [comment]
@rroller Any possible update on getting these changes merged into the main branch?

I am not well-versed in git, but I manually made the changes found in [this commit](https://github.com/rroller/dahua/pull/315/commits/9816e07c699b060807a8487fda93c6c3eb728424) to the __init__.py and light.py files from the most recent release (0.9.59) and can confirm that my ASH26-W floodlight turns on and off as expected.

Prior to making those changes, I still got the errors that I outlined [here](https://github.com/rroller/dahua/issues/151#issuecomment-2161133748).

**Comment by rroller** [comment]
done. released in https://github.com/rroller/dahua/releases/tag/0.9.60

**Pull request closed without merging**